course/versioning.py

```diff
@@ -183,11 +185,18 @@ class Meta:
             "enrollment_required_email_suffix",
             "from_email",
             "notify_email",
+            "force_lang",
             )
         widgets = {
                 "start_date": DateTimePicker(options={"format": "YYYY-MM-DD"}),
                 "end_date": DateTimePicker(options={"format": "YYYY-MM-DD"}),
                 }
+        from django.conf import settings
+        if getattr(settings, "RELATE_ENABLE_COURSE_SPECIFIC_LANG", False):
```

Comment:
Do we need `RELATE_ENABLE_COURSE_SPECIFIC_LANG`? It just adds code complexity, and instructors that don't want it can just leave it alone.

Reply:
Ok, that really simplified the jobs.